### ISSUE TITLE
refinementList issue when upgrading from 0.5.0 to 1 instantsearch.js?

### ISSUE THREAD

**Issue by bvacarmen** [algolia/instantsearch.js#773]
Trying to upgrade instantsearch.js from 0.5.0 to 1 on Algolia shopify app. 

I get this error in the console:
```
Uncaught Error: Usage:
refinementList({
  container,
  attributeName,
  [ operator='or' ],
  [ sortBy=['count:desc'] ],
  [ limit=1000 ],
  [ cssClasses.{root,header,body,footer,list,item,active,label,checkbox,count}],
  [ templates.{header,item,footer} ],
  [ transformData ],
  [ autoHideContainer=true ]
})
```

Which seems to be from the refinement-list.js file. 

My algolia facets are not current using the .refinementList function (it's whatever came packaged with the most current Algolia shopify app) but I'm guessing that it should? for the facets to work? 

Can anyone point me in the right direction?

**Comment by vvo** [comment]
Hi @bvacarmen indeed instantsearch.js 0.5.0 was never "officially" released nor supported, and switching from 0.5.0 to 1.x.x cannot be done without any modification in your code.

I think @Jerskouille will be able to help more on the shopify + instantsearch.js support. He will reply here, thanks!

**Comment by Jerskouille** [comment]
This this issue is not directly related to `instantsearch.js`, I'll answer via e-mail.

[But just in order to have an answer to the issue, I believe it's the `attributeName` which replaced `facetName` in a release before v1].
